src/com/theironyard/novauc/MonkeyTrouble.java

```diff
@@ -0,0 +1,21 @@
+package com.theironyard.novauc;
+
+//We have two monkeys, a and b, and the parameters aSmile and bSmile indicate if each is smiling.
+        //We are in trouble if they are both smiling or if neither of them is smiling. Return true if we are in trouble.
+
+/**
+ * Created by ANVIL_OCTOPUS on 4/9/17.
+ */
+public class MonkeyTrouble {
+
+    public boolean monkeyTrouble(boolean aSmile, boolean bSmile) {
+        if (aSmile && bSmile) {
```

Comment:
You have to write tests for these.  Coding bat gives you test cases above every problem, so you don't have to figure out the scenarios.  But you do have to write the test methods using JUnit.